lib/source/layouts/_search.erb

```diff
@@ -1,16 +1,22 @@
 <% if config[:tech_docs][:enable_search] %>
 <div class="search" data-module="search">
-  <form action="https://www.google.co.uk/search" method="get" role="search">
+  <form action="https://www.google.co.uk/search" method="get" role="search" class="search__form govuk-!-margin-bottom-4">
     <input type="hidden" name="as_sitesearch" value="<%= config[:tech_docs][:host] %>"/>
-    <label class="govuk-label search__label" for="search">
+    <label class="govuk-label search__label" for="search" aria-hidden="true">
       Search (via Google)
     </label>
-    <input class="govuk-input govuk-!-margin-bottom-4" id="search" name="q" type="text" aria-controls="search-results" placeholder="Search">
+    <input
+      type="text"
+      id="search" name="q"
+      class="govuk-input govuk-!-margin-bottom-0 search__input"
+      aria-controls="search-results"
+      placeholder="Search">
+    <button type="submit" class="govuk-button search__button">Search</button>
```

Comment:
I'm not sure what the intended behaviour is for this button, but clicking on it without typing anything in first makes the search index panel appear like this:

![Screenshot 2021-03-08 at 13 14 02](https://user-images.githubusercontent.com/861310/110326248-4d523f80-8010-11eb-87ea-0fc12bab9dc4.png)

It should instead probably show the 'no results' state.

![Screenshot 2021-03-08 at 13 15 40](https://user-images.githubusercontent.com/861310/110326389-825e9200-8010-11eb-8946-e3c6d625e8f2.png)

Reply:
Hmmmm. The behaviour when you click the button without typing anything in seems to be the same as the behaviour when you press Enter with the empty field in focus on the [current version of tech docs](https://docs.publishing.service.gov.uk/). This wasn't flagged as an issue in the report, that I could tell. 😕  But maybe it should have been?

Reply:
Alright @andysellick  I think this should be fixed now